doc/UsersGuide/source/solving.rst

```diff
@@ -73,6 +73,29 @@ solver and furthermore it has the following IDA specific flags:
 :ref:`idaNonLinConvCoef <simflag-idaNonLinConvCoef>`,
 :ref:`idaMaxErrorTestFails <simflag-idaMaxErrorTestFails>`.
 
+
+.. _sundials_cvode :
+
+CVODE
+~~~~~
+
+The CVODE solver is part of sundials: SUite of Nonlinear and
+DIfferential/ALgebraic equation Solvers :cite:`Hindmarsh:2005`.
+CVODE solves initial value problems for ordinary differential equation (ODE)
+systems with variable-order, variable-step multistep methods.
+
+In OpenModelica CVODE uses a combination of Backward Differentiation
+Formulas (BDFs), with order varying between 1 and 5, and a Modified Newton
+iteration with fixed Jacobian as default.
+This setting is advised for stiff problems and therefore chooses by default.
```

Comment:
Chosen by default?

Reply:
If you just use `-s=cvode` it will use BDF as linear multistep method and Newton as non-linear solver.
May be I should reword it to: `This setting is advised for stiff problems and enabled per default.`

Reply:
If it's the OpenModelica nonlinear solver (unrelated to cvode internals), I guess that should be clarified as well.

Reply:
I can add that as well